packages/pieces/community/framework/src/lib/validators/validators.ts

```diff
@@ -258,7 +258,7 @@ class Validators {
   static integer: TypedValidatorFn<ValidationInputType.NUMBER> = {
     type: ValidationInputType.NUMBER,
     fn: (property, processedValue, userInput) => {
-      if (isInteger(processedValue)) {
+      if (!isInteger(processedValue)) {
```

Comment:
Ouch good catch, can you bump the package.json for framework.

Reply:
hello  @abuaboud  I will make the change to package.json.

Sry I forgot.